### ISSUE TITLE
Crashes on startup

### ISSUE THREAD

**Issue by Wandelstock** [Kunzisoft/KeePassDX#774]
**Describe the bug**
Tapping keepas files opens app, instead it crashes

**To Reproduce**
see above

** Keepass Database **
 - Created Windows KeePass
 - Version: [e.g. 2]
 - Location: gdrive
 - Size: small
 - Contains attachment: no

**KeePassDX (please complete the following information):**
 - Version: 2.9 playstore
 - Build: pro
 - Language: english

**Android (please complete the following information):**
 - only on android 6
 ]

**Additional context**
 - Browser for Autofill: [e.g. Chrome version X]

**Comment by J-Jamet** [comment]
Thank you @Wandelstock for the feedback. I don't understand why this only happens with Android 6. I'll check it out and get back to you.

**Comment by Wandelstock** [comment]
tried it with no database and it crashes too. I cant send a log.

**Comment by J-Jamet** [comment]
```
11-02 20:23:47.362 4916-4916/com.kunzisoft.keepass.pro E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.kunzisoft.keepass.pro, PID: 4916
    java.lang.RuntimeException: Unable to start activity ComponentInfo{com.kunzisoft.keepass.pro/com.kunzisoft.keepass.activities.PasswordActivity}: android.view.InflateException: Binary XML file line #64: Binary XML file line #64: Error inflating class ImageView
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2416)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2476)
        at android.app.ActivityThread.-wrap11(ActivityThread.java)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1344)
        at android.os.Handler.dispatchMessage(Handler.java:102)
        at android.os.Looper.loop(Looper.java:148)
        at android.app.ActivityThread.main(ActivityThread.java:5417)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
     Caused by: android.view.InflateException: Binary XML file line #64: Binary XML file line #64: Error inflating class ImageView
        at android.view.LayoutInflater.inflate(LayoutInflater.java:539)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:423)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:374)
        at androidx.appcompat.app.AppCompatDelegateImpl.setContentView(AppCompatDelegateImpl.java:696)
        at androidx.appcompat.app.AppCompatActivity.setContentView(AppCompatActivity.java:170)
        at com.kunzisoft.keepass.activities.PasswordActivity.onCreate(PasswordActivity.kt:122)
        at android.app.Activity.performCreate(Activity.java:6237)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1107)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2369)
```

It comes from the XML image, but only this android version, that's a mystery. I will correct this problem for the next version quickly.

**Comment by J-Jamet** [comment]
The problem stems from the gradient color of the logo. :/

**Comment by J-Jamet** [comment]
It's resolved, I fix some bugs and I release version 2.9.1
